.lintstagedrc

```diff
@@ -1,5 +1,6 @@
 {
   "*.{js,ts,tsx}": [
+    "bundlesize --config bundlesize.config.js",
```

Comment:
These commands would run the bundlesize command multiple times for each file change. I think this may work better as a separate command in the `precommit` hook.